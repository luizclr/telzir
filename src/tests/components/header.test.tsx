import React from "react";
import Header from "../../components/header";
import { shallow } from "enzyme";

describe("<Header />", () => {
  it("should render", () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
