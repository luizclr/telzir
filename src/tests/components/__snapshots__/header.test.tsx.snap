// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render 1`] = `
<div
  className="header"
>
  <img
    alt="logo"
    src="emoji.svg"
  />
  <h1
    className="title"
  >
    Tel
    <span>
      z
    </span>
    ir
  </h1>
</div>
`;
